superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -218,6 +234,10 @@ class Header extends React.PureComponent {
         interval,
         chartCount: affectedCharts.length,
       });
+      this.props.addWarningToast(
+        t(`This dashboard will force refresh at every ${intervalMessage}`),
```

Comment:
This toast pops up when the dashboard starts refreshing right? It's a bit weird to say it `will force refresh` if it's currently force refreshing. Maybe the wording should be `This dashboard is currently force refreshing; the next force refresh will be in ${intervalMessage}.`?

cc @sylvia-tomiyama, @zuzana-vej for copy here